src/shadowsocks_config.ts

```diff
@@ -320,7 +316,7 @@ export const SIP002_URI = {
 
   stringify: (config: Config) => {
     const {host, port, method, password, tag, extra} = config;
-    const userInfo = Base64.encode(`${method.data}:${password.data}`);
+    const userInfo = Base64.encodeURI(`${method.data}:${password.data}`);
```

Comment:
The [js-base64 docs](https://github.com/dankogai/js-base64#synopsis) state that `encodeURI` uses "-" as padding for URIs, but I'm seeing that they actually follow the [RFC recommendation](https://tools.ietf.org/html/rfc4648#section-5) to omit it for known length data.

Can we please add a comment to this end?

Reply:
added a pr upstream to add more example of when they do and don't add padding dashes. https://github.com/dankogai/js-base64/pull/139